### PR TITLE
update vouch config

### PR DESCRIPTION
**Pull request by dB2510**
Updates vouch config to add timeout to the required endpoints. Also adds a common charon-env extension field config to be used for each charon node.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # in docker-compose.yml. Rename this file to `.env` and then uncomment and set any variable below.
 
 # Supported ethereum network (for ex: mainnet, goerli, sepolia, ropsten, gnosis) .
-#ETH2_NETWORK=
```
Comment (corverroos) [inline .env.sample:5]: rather follow the cdvn repo pattern that uses defaults so using .env is optional not required.

File: vouch/vouch.sample.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ strategies:
     # beacon-node-addresses are the addresses of beacon nodes to use for this strategy.
     beacon-node-addresses:
       -
+    timeout: 4s
```
Comment (corverroos) [inline vouch/vouch.sample.yml:33]: vmock uses 10s timeout

File: .env.sample (hunk 1 above)
Comment (dB2510) [reply .env.sample]: done

File: vouch/vouch.sample.yml (hunk 2 above)
Comment (dB2510) [reply vouch/vouch.sample.yml]: sure

File: docker-compose.yml [hunk 3]
```diff
@@ -5,24 +5,25 @@ x-node-base:
...
-    - CHARON_MONITORING_ADDRESS=0.0.0.0:3620
-    - CHARON_JAEGER_ADDRESS=${CHARON_JAEGER_ADDRESS-jaeger:6831} # Overriding to empty address allowed
-    - CHARON_JAEGER_SERVICE=charon
-  env_file: .env
   networks: [ cluster ]
   depends_on: [ bootnode ]
   volumes:
     - ./.charon:/opt/charon/.charon/
 
+x-charon-env:
```
Comment (xenowits) [inline docker-compose.yml:13]: `x-node-env` ?

**Review by xenowits — APPROVED** [review]

**Review by corverroos — APPROVED** [review]

**Pull request merged**